flow-typed/types/playback-config.js

```diff
@@ -2,6 +2,8 @@
 declare type PKPlaybackConfigObject = {
   audioLanguage: string,
   textLanguage: string,
+  extensionAudioLanguage: string,
```

Comment:
I would use "secondaryXLanguage" or "additionalXLanguage"
"extensionXLanguage" is not clear enough

Reply:
done